[2024-10-08T01:52:43.142+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T01:52:43.155+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tab_soccer_statics.fact_matches manual__2024-10-08T01:51:48.031379+00:00 [queued]>
[2024-10-08T01:52:43.160+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tab_soccer_statics.fact_matches manual__2024-10-08T01:51:48.031379+00:00 [queued]>
[2024-10-08T01:52:43.160+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-08T01:52:43.174+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fact_matches> on 2024-10-08 01:51:48.031379+00:00
[2024-10-08T01:52:43.185+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=668) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T01:52:43.187+0000] {standard_task_runner.py:72} INFO - Started process 681 to run task
[2024-10-08T01:52:43.187+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tab_soccer_statics', 'fact_matches', 'manual__2024-10-08T01:51:48.031379+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/tab_soccer_statics.py', '--cfg-path', '/tmp/tmpap4e24bc']
[2024-10-08T01:52:43.189+0000] {standard_task_runner.py:105} INFO - Job 105: Subtask fact_matches
[2024-10-08T01:52:43.224+0000] {task_command.py:467} INFO - Running <TaskInstance: tab_soccer_statics.fact_matches manual__2024-10-08T01:51:48.031379+00:00 [running]> on host f74ce347067d
[2024-10-08T01:52:43.293+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='daniel.cristancho' AIRFLOW_CTX_DAG_ID='tab_soccer_statics' AIRFLOW_CTX_TASK_ID='fact_matches' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T01:51:48.031379+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T01:51:48.031379+00:00'
[2024-10-08T01:52:43.294+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T01:52:43.339+0000] {warnings.py:112} WARNING - /opt/***/dags/tab_soccer_statics.py:69: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  data_extract = pd.read_json(data_extract)

[2024-10-08T01:52:45.111+0000] {fact_matches.py:31} INFO - Start extract loaded data
[2024-10-08T01:52:45.112+0000] {fact_matches.py:175} INFO - Staring data extraction for dim_teams
[2024-10-08T01:52:51.630+0000] {fact_matches.py:179} INFO - 261 rows extracted from dim_teams
[2024-10-08T01:52:51.631+0000] {fact_matches.py:180} INFO - Data extraction successfully finished
[2024-10-08T01:52:51.631+0000] {fact_matches.py:175} INFO - Staring data extraction for dim_league
[2024-10-08T01:52:52.066+0000] {fact_matches.py:179} INFO - 9 rows extracted from dim_league
[2024-10-08T01:52:52.066+0000] {fact_matches.py:180} INFO - Data extraction successfully finished
[2024-10-08T01:52:52.066+0000] {fact_matches.py:175} INFO - Staring data extraction for fact_matches
[2024-10-08T01:52:52.279+0000] {fact_matches.py:179} INFO - 2153 rows extracted from fact_matches
[2024-10-08T01:52:52.280+0000] {fact_matches.py:180} INFO - Data extraction successfully finished
[2024-10-08T01:52:52.280+0000] {fact_matches.py:42} INFO - Start transform data
[2024-10-08T01:52:52.348+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:120: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["uuid"] = result_append.apply(lambda _: uuid.uuid4(), axis=1)

[2024-10-08T01:52:52.351+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:121: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["created_at"] = pd.to_datetime(

[2024-10-08T01:52:52.352+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:124: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["updated_at"] = pd.to_datetime(

[2024-10-08T01:52:52.352+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:127: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["id_match"] = result_append["id_match"].astype("Int64")

[2024-10-08T01:52:52.353+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:128: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["home_goals"] = result_append["home_goals"].astype("Int64")

[2024-10-08T01:52:52.354+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:129: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["away_goals"] = result_append["away_goals"].astype("Int64")

[2024-10-08T01:52:52.354+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:130: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["home_goals_half_time"] = result_append[

[2024-10-08T01:52:52.355+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:133: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["away_goals_half_time"] = result_append[

[2024-10-08T01:52:52.356+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:136: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["date_match"] = result_append["date_match"].astype("Int64")

[2024-10-08T01:52:52.357+0000] {warnings.py:112} WARNING - /opt/***/dags/src/soccer_statics/fact_matches.py:137: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  result_append["hour_match"] = result_append["hour_match"].astype("Int64")

[2024-10-08T01:52:52.357+0000] {fact_matches.py:45} INFO - Start load new data
[2024-10-08T01:52:52.358+0000] {fact_matches.py:160} INFO - writing fact_matches table
[2024-10-08T01:52:55.095+0000] {fact_matches.py:164} INFO - 1592 rows load in fact_matches
[2024-10-08T01:52:55.095+0000] {fact_matches.py:165} INFO - loading operation finished successfully for fact_matches
[2024-10-08T01:52:55.209+0000] {logging_mixin.py:190} INFO - ****************** Success process update fact matches
[2024-10-08T01:52:55.214+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-08T01:52:55.225+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T01:52:55.225+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tab_soccer_statics, task_id=fact_matches, run_id=manual__2024-10-08T01:51:48.031379+00:00, execution_date=20241008T015148, start_date=20241008T015243, end_date=20241008T015255
[2024-10-08T01:52:55.275+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-08T01:52:55.289+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-08T01:52:55.291+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
